data/__init__.py

```diff
@@ -61,11 +61,20 @@ def create_dataloader(opt, rank, dataset, batch_size):
 
 
 def create_dataset_temporal(opt, phase):
-    dataset_class = find_dataset_using_name("temporal_labeled_mask_online")
+    dataset_class = find_dataset_using_name(
```

Comment:
I believe this function needs to be change so that either `temporal_labeled_mask_online` or `self_supervised_temporal_labeled_mask_online` is selected based on whether `cut` or `palette` is running.